node/Cargo.toml

```diff
@@ -3,103 +3,102 @@ name = "zeitgeist"
+session-keys-primitives = { tag = "v0.27.2-a", default-features = false, git = "https://github.com/zeitgeistpm/moonbeam", optional = true }
+sp-keystore = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate", optional = true }
+substrate-prometheus-endpoint = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate", optional = true }
 
 # Polkadot
 
-polkadot-cli = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-parachain = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-primitives = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-service = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
-polkadot-test-service = { branch = "release-v0.9.26", git = "https://github.com/paritytech/polkadot", optional = true }
+polkadot-cli = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-parachain = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-primitives = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-service = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
+polkadot-test-service = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/polkadot", optional = true }
 
 # Standalone
 
-sc-consensus-aura = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
-sc-finality-grandpa = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
-sp-consensus-aura = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
-sp-finality-grandpa = { branch = "polkadot-v0.9.26", git = "https://github.com/paritytech/substrate" }
+sc-consensus-aura = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
+sc-finality-grandpa = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
+sp-consensus-aura = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
+sp-finality-grandpa = { branch = "moonbeam-polkadot-v0.9.29", git = "https://github.com/zeitgeistpm/substrate" }
 
 # Utility
 cfg-if = { version = "1.0.0" }
 clap = { version = "3.2.6", features = ["derive"] }
 hex-literal = { version = "0.3.4" }
-jsonrpsee = { version = "0.14.0", features = ["server"] }
+jsonrpsee = { version = "0.15.1", features = ["server"] }
 log = { optional = true, version = "0.4.17" }
-# TODO(#865): Remove in future Polkadot release
-tracing-core = "=0.1.26"
```

Comment:
In this branch there are no migrations yet, it will be there once I merge main into this branch. But it does print lots of INFO and DEBUG traces so I think we are good now even without above explicit pinning of the dependency.